data/json/items/gun/223.json

```diff
@@ -325,6 +325,52 @@
     "modes": [ [ "DEFAULT", "semi-auto", 1 ], [ "BURST", "3rd burst", 3 ] ],
     "magazines": [ [ "223", [ "stanag30", "stanag50", "survivor223mag" ] ] ]
   },
+  {
+    "id": "oa93",
+    "copy-from": "rifle_semi",
+    "looks_like": "skorpion_61",
+    "type": "GUN",
+    "name": { "str": "OA-93" },
+    "description": "An AR-15 derivative pistol manufactured by Olympic Arms in the nineties. The main difference compared to the AR-15 is that the recoil spring has been moved to the top of the gun, circumventing the necessity for a solid buttstock.",
+    "weight": "2023 g",
```

Comment:
Which weight are you using for this?

Reply:
Second source for the pistol (Length/Weight: 17.00in. / 4.46lbs.).
The first source only states the OA-93 carbine weight.